### PR TITLE
CLM & Anomalies in Gutter

### PR DESCRIPTION
**Pull request by CalvinAllen**

### CODE REVIEW

File: vs/src/CodeStream.VisualStudio.Shared/WebViewRouter.cs [hunk 1]
```diff
@@ -195,7 +195,8 @@ public async System.Threading.Tasks.Task HandleAsync(WindowEventArgs e)
 								{
 									using (var scope = _browserService.CreateScope(message))
 									{
-										await CodeLevelMetricsCallbackService.RefreshAllCodeLensDataPointsAsync();
+										// TODO CODE LEVEL METRICS?
+										//await CodeLevelMetricsCallbackService.RefreshAllCodeLensDataPointsAsync();
```
Comment (bcanzanella) [inline vs/src/CodeStream.VisualStudio.Shared/WebViewRouter.cs:199]: @CalvinAllen this is ok commented out?

Comment (CalvinAllen) [reply vs/src/CodeStream.VisualStudio.Shared/WebViewRouter.cs:199]: @bcanzanella I don't know exactly what it should do now (and forgot about it actually) - any thoughts?

File: vs/src/CodeStream.VisualStudio.Core/Models/AverageDurationResponse.cs [hunk 2]
```diff
@@ -18,5 +18,8 @@ public class AverageDurationResponse
 
 		[JsonProperty("metricTimesliceName", NullValueHandling = NullValueHandling.Ignore)]
 		public string MetricTimesliceName { get; set; }
+
+		[JsonProperty("anomaly", NullValueHandling = NullValueHandling.Ignore)]
```
Comment (bcanzanella) [inline vs/src/CodeStream.VisualStudio.Core/Models/AverageDurationResponse.cs:22]: does this need to be `observabilityAnomaly`?

Comment (CalvinAllen) [reply vs/src/CodeStream.VisualStudio.Core/Models/AverageDurationResponse.cs:22]: Nope, the name doesn't match the type name in this one.

File: vs/src/CodeStream.VisualStudio.Shared/WebViewRouter.cs (hunk 1 above)
Comment (CalvinAllen) [reply vs/src/CodeStream.VisualStudio.Shared/WebViewRouter.cs]: Okay, this is resolved - ready for final review

**Review by bcanzanella — APPROVED** [review]

**Comment by CalvinAllen** [comment]
Merge POST-UID release

**Comment by bcanzanella** [comment]
@CalvinAllen can you check on these conflicts?

**Comment by CalvinAllen** [comment]
@bcanzanella Fixed; ready to roll.

**Comment by bcanzanella** [comment]
let's wait until or after Jan 12

**Pull request merged**